src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs

```diff
@@ -233,6 +233,11 @@ private static string FetchRequestId(IRequestContext requestContext, IResponseCo
         {
             activity.SetTag(AWSSemanticConventions.AttributeAWSServiceName, service);
             activity.SetTag(AWSSemanticConventions.AttributeAWSOperationName, operation);
+
+            // Follow: https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/instrumentation/aws-sdk/#common-attributes
```

Comment:
I would put here link to exact version (link with tag to GitHub semantic convention repository) rather than OTel.io

Reply:
```suggestion
            // Follow: https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/cloud-providers/aws-sdk.md#common-attributes
```